Formula/form.rb

```diff
@@ -5,6 +5,11 @@ class Form < Formula
   sha256 "f1f512dc34fe9bbd6b19f2dfef05fcb9912dfb43c8368a75b796ec472ee8bbce"
   license "GPL-3.0-or-later"
 
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

Comment:
```suggestion
    strategy :github_latest
```

The `stable` URL is a release asset, so we should use the `GithubLatest` strategy here.